### PR TITLE
add asManyPotatosAsYouWant

### PR DESCRIPTION
**Pull request by kingdevnl**
Create thePotato instance for the Potato class.
So we can acces the Loved potato more Easy!

### CODE REVIEW

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by MiniDigger** [comment]
Why would we only have one potato when we could have unlimited due to the propagate method? 
This self imposed limitation of availability of potatoes doesn't make any sense!

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by MiniDigger** [comment]
Don't you think upvoting once is enough? Also, use the GitHub reactions, no need to send out email notifications.

**Comment by kingdevnl** [comment]
Edit: Create thePotato instance and Add asManyPotatosAsYouWant

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by MiniDigger** [comment]
Your asManyPotatosAsYouWant method is a joke, right? Why would you want a list which contains the same reference multiple times?
It still doesn't make any sense to have the potato as a Singleton. The condiment would always have the same expire state for example. 
Can you provide arguments why making the potato a Singleton (and thus limiting us to a single potato instance!) is a good idea?

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by kingdevnl** [comment]
You make some kind of a point btw!,
And jep was meant as a joke Hahaha

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by kingdevnl** [comment]
Ok so its changed up so now it isnt a singleton anymore
+ It creates a new Potato instance every time in asManyPotatosAsYouWant

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by MathisSanders** [comment]
Vouch! 👍

**Comment by JanTuck** [comment]
Vouch! 👍🏼

**Comment by MathisSanders** [comment]
Vouch! 👍

File: src/main/java/org/drtshock/Potato.java [hunk 1]
```diff
@@ -11,6 +11,9 @@
  */
 public class Potato implements Tuber {
 
+
+
+
```
Comment (drtshock) [inline src/main/java/org/drtshock/Potato.java]: lots of space?

Comment (lol768) [reply src/main/java/org/drtshock/Potato.java]: Vouch!

Comment (MathisSanders) [reply src/main/java/org/drtshock/Potato.java]: Vouch! 👍

**Comment by MathisSanders** [comment]
Vouch! 👍

File: src/main/java/org/drtshock/Potato.java (hunk 1 above)
Comment (kingdevnl) [reply src/main/java/org/drtshock/Potato.java]: Wuut, didnt even see that o.0

Comment (MathisSanders) [reply src/main/java/org/drtshock/Potato.java]: Vouch! 👍

**Pull request closed without merging**